app.py

```diff
@@ -58,17 +59,28 @@ def edit(item):
 
     return render_template("edit_inventory.html", item=item)
 
-@app.route("/search", methods=["GET", "POST"])
-def search():
-    if request.method == "POST":
-
-        # something
-
-        # redirect
-        return redirect(url_for("search"))
-    
-    # return the render template
-
+@app.route("/recipes", methods=["GET"])
+def recipes():
```

Comment:
May need a case for "POST" like what if the user wants to select the items that they would like to use. Look at prior functions to get a better understanding or ask me later to get help.